README.md

```diff
@@ -27,6 +27,7 @@ A list of every a11y concern Checka11y.css will check for and highlight:
 - Checks `<li>` is the **only** child of `<ol>` and `<ul>`
 - Checks `<nav>` only exists once on any page
 - Missing `lang` on `<html>`
+- Reminder that `<a target="_blank">` link's behavior should be communicated in a way that is apparent to all users.
```

Comment:
Let's make this a little less technical talk 😊

"Warns about links that open in a new window"

Does the above sound good?

The consumer of Checka11y will be able to see more thorough details on https://checka11y.jackdomleo.dev/example soon.